docker/clone-and-preflight.ts

```diff
@@ -71,8 +71,10 @@ if (projectUsesPostgresql) {
   process.env.PGPASSWORD = 'project_to_check';
 
   // Create directory for PostgreSQL socket
-  await executeCommand('mkdir /run/postgresql');
-  await executeCommand('chown postgres:postgres /run/postgresql');
+  await executeCommand('mkdir -p /postgres-volume/run/postgresql/data');
+  await executeCommand(
+    'chown -R postgres:postgres /postgres-volume/run/postgresql',
+  );
```

Comment:
Used `-p` to make sure that the directory gets created but when the directory already exists, it won't throw an error, and if any parent directories are missing, they will be created.

With the `-R` option, I make sure that the ownership change is applied to all files and subdirectories within `/postgres-volume/run/postgresql`